README.md

```diff
@@ -84,20 +84,20 @@ cd libtrixi-julia
     --p4est-library <p4est_install_directory>/lib/libp4est.so
     <install_directory>
 ```
-
 Use `libtrixi-init-julia -h` to get help.
-When running a program that uses libtrixi, make sure to set up the `JULIA_DEPOT_PATH`
-environment variable to point to the `<julia-depot>` folder reported. In your code, pass
-the path to the `libtrixi-julia` directory to `trixi_initialize`, see the code of the
-examples.
+
+In your code, pass the path to the `libtrixi-julia` directory to `trixi_initialize`,
+see the code of the examples. If you did not modify the default value for the Julia depot
+when calling `libtrixi-init-julia`, libtrixi will find it automatically.
+Otherwise, when running a program that uses libtrixi, you need to make sure to set the
+`JULIA_DEPOT_PATH` environment variable to point to the `<julia-depot>` folder reported. 
```

Comment:
Would you want to mention the other option, by supplying the depot path to `trixi_initialize`?

Reply:
No, this should imho go into the API docs (whenever they come :-))